index.js

```diff
@@ -425,85 +457,165 @@ module.exports = {
     // assertion that the variable exists in given scope and the actual
     // assignment happens later. Here, it makes sense to save space and combine
     // all declared variables into one statement.
-    "one-var": ["error", { initialized: "never", uninitialized: "always" }],
-    "operator-assignment": ["error", "always"],
-    "operator-linebreak": ["error", "before"],
-    "padded-blocks": ["error", "never"],
-    "quote-props": ["error", "consistent"],
-    "quotes": ["error", "double"],
-    "require-jsdoc": "off",
-    "semi-spacing": ["error", { before: false, after: true }],
-    "semi": ["error", "always"],
+    "one-var": [ "error", { initialized: "never", uninitialized: "always" }],
+
+    // Disabled because `one-var` doesn't allow multiple initialized variable
+    // declarations in one `let`/`const`/`var` statement and it doesn't make
+    // sense to put each uninitialized variable on a separate line.
+    "one-var-declaration-per-line": "off",
+    "operator-assignment": [ "error", "always" ],
+    "operator-linebreak": [ "error", "before" ],
+    "padded-blocks": [ "error", "never" ],
+    "padding-line-between-statements": "off",
+    "prefer-exponentiation-operator": "off",
+    "prefer-object-spread": "error",
+    "quote-props": [ "error", "consistent" ],
+    "quotes": [ "error", "double" ],
```

Comment:
Adding `"avoidEscape": true` should significantly increase readability of some pieces of codegenerator.